config/routes.rb

```diff
@@ -7,4 +7,5 @@
   root "jobs#index"
 
   get "app_inspector", to: "app_inspector#index"
+  post 'replace_app_inspector', :to => 'app_inspector#replace_stream', :as => :replace_app_inspector
```

Comment:
It's not a `form` we're using HTTP to POST to.  The existing controller#index should be able to respond to `turbo_stream` request formats.  Like a client sending an `accept: json` header to get json data, we're sending an `accept: xhtml/tubo-stream` (or similar).


This `respond_to` is so simple you may not even need it if you just create the `index.turbo_stream.erb` Rails may just do it.

```ruby
  def index
    @app_names = Job.all_names
    @app_cpus = Job.app_cpus

     respond_to do |format|
         format.html
         format.turbo_stream       
     end
  end


# [app/views/app_inspector/index.turbo_stream.erb
<%= turbo_stream.replace "app_inspector_histogram" do %>
  <%= render partial: 'histogram', locals: { graph_data: @app_cpus['graph_data'], bin_size: @app_cpus['bin_size'] }
<% end %>